### ISSUE TITLE
[Bug] Slash forward issue

### ISSUE THREAD

**Issue by Chavithra** [OpenBB-finance/OpenBBTerminal#2042]
**Describe the bug**
Slash forward seems to be bugged.

**To Reproduce**
```
2022 Jul 04, 10:03 (🦋) / $ stocks / disc
usage: terminal {update,about,keys,settings,featflags,exe,stocks,economy,crypto,portfolio,forex,etf,reports,dashboards,funds,alternative,econometrics,cls,home,h,?,help,q,quit,..,exit,r,reset,support}
terminal: error: argument cmd: invalid choice: 'disc' (choose from 'update', 'about', 'keys', 'settings', 'featflags', 'exe', 'stocks', 'economy', 'crypto', 'portfolio', 'forex', 'etf', 'reports', 'dashboards', 'funds', 'alternative', 'econometrics', 'cls', 'home', 
'h', '?', 'help', 'q', 'quit', '..', 'exit', 'r', 'reset', 'support')

The command ' disc' doesn't exist on the / menu
```

- OS: windows
- Python version : 3.8.7

**Comment by jmaslek** [comment]
This should get fixed by #2058  ?

**Comment by DidierRLopes** [comment]
Yes, this was the issue! @jmaslek
